src/main/java/com/diamondfire/helpbot/bot/command/impl/other/fun/GarfieldCommand.java

```diff
@@ -40,23 +42,16 @@ public Permission getPermission() {
     @Override
     public void run(CommandEvent event) {
         EmbedBuilder builder = new EmbedBuilder();
-        try {
-            URL url = new URL("https://labscore.vercel.app/v2/garfield");
-            try (BufferedReader in = new BufferedReader(new InputStreamReader(url.openStream()))) {
-                String link = JsonParser.parseString(in.readLine()).getAsJsonObject().get("link").getAsString();
-                
-                if (link == null) {
-                    throw new IOException();
-                } else {
-                    builder.setTitle("Garfield Comic");
-                    builder.setImage(link);
-                    builder.setColor(new Color(252, 166, 28));
-                }
-            }
-        } catch (Exception e) {
-            builder.setTitle(":rotating_light: API BROKE :rotating_light:");
-            builder.setDescription("DM: <@223518178100248576>\nPING: <@223518178100248576>");
-        }
+
+        Date randomDate = new Date(ThreadLocalRandom.current()
+            .nextLong(MIN, MAX);
+
+        SimpleDateFormat formatter = new SimpleDateFormat("yyyy/yyyy-MM-dd");
```

Comment:
Forgot this

Reply:
Ah fuck

Reply:
Seems to be there? Not sure what you’re on aboht

Reply:
the simple date formatter isn't a static field